src/Main.ts

```diff
@@ -0,0 +1,108 @@
+import Observer from "./Observer";
```

Comment:
this file really should be `main.ts` since it's not a Class file, and is consistent with other library naming conventions

Reply:
OK, what you said makes sense, let me change it to lowercase.

Reply:
No, how can Observer replace the entry file of the main.ts plugin?

Reply:
sorry, there's no way to comment on a file itself. I was just saying that this file `Main.ts` should be renamed to `main.ts` and then update accordingly in the package.json, per my other comment